rosdep/base.yaml

```diff
@@ -1400,13 +1401,15 @@ libdrm-dev:
       packages: [x11-base/nouveau-drm]
   ubuntu: [libdrm-dev]
 libesd0-dev:
+  debian: [libesd0-dev]
   fedora: [esound-devel]
   ubuntu: [libesd0-dev]
 libespeak-dev:
   debian: [libespeak-dev]
   fedora: [espeak-devel]
   ubuntu: [libespeak-dev]
 libestools-dev:
+  debian: [libestools2.1-dev]
```

Comment:
In this case we will need to make a subcategory for jessie and for stretch referring to the right non-virtual packages